pubsub-adapter-compose.yml

```diff
@@ -6,4 +6,11 @@ services:
     environment:
```

Comment:
do we need a separate compose file here, or can we have just the one compose file and use commands like ```docker-compose -f <compose.yml> up -d <individual-service>``` to bring up individual services